specs/configure_monitoring.adoc

```diff
@@ -0,0 +1,418 @@
+// vim: tw=79
+
+= Configuring collectd
+
+== Introduction
+
+Tendrl's monitoring stack uses the following components:
+
+* collectd
+  ** A daemon process that resides on tendrl managed nodes, collects
+     performance statistics and pushes them to time-series db for maintaining
+     historic data.
+  ** Everything in collectd is achieved through plugins.
+  ** Collectd activates those plugins that are loaded and/or configured in
+     collectd’s conf files.
+  ** Changes in conf files will be effected in collectd only on restart of
+     the collectd daemon.
+
+* graphite
+  ** Used by tendrl monitoring stack as time series db
+  ** It consists of three software components:
+    *** carbon - a high-performance service that listens for time-series data
+    *** whisper - a simple database library for storing time-series data
+    *** graphite-web - Graphite's user interface & API for rendering graphs
+        and dashboards
+  ** Metrics get fed into the stack via the Carbon service, which writes data
+     out to Whisper databases for long-term storage.
+
+== Problem Description
+
+Tendrl needs to configure collectd for it to start collecting the performance
+statistics at regular intervals of time. This process of configuring collectd
+should require no/minimal user intervention.
```

Comment:
Don't we need to explicitly state that we are solving historic time-series data problem here as opposed to reporting that something just happened immediately (which requires to deliver such message in given time range, and which is not being part of this specification)?

I would like to make this clear immediately in the description (if this is the case).